scrapper.py

```diff
@@ -89,9 +94,39 @@ def validate_config(crawler_path):
     """
     Validates given config
     """
-    pass
+    with open(crawler_path, 'r') as crawler_config_file:
+        crawler_config = json.load(crawler_config_file)
+
+    is_crawler_config_ok = True
+    if not isinstance(crawler_config, dict) or \
+            "base_urls" not in crawler_config or "total_articles_to_find_and_parse" not in crawler_config \
+            or "max_number_articles_to_get_from_one_seed" not in crawler_config:
+        is_crawler_config_ok = False
+        raise UnknownConfigError
+
+    elif not isinstance(crawler_config["base_urls"], list) or \
+            not (isinstance(url, str) for url in crawler_config["base_urls"]):
+        is_crawler_config_ok = False
+        raise IncorrectURLError
+
+    elif not isinstance(crawler_config["total_articles_to_find_and_parse"], int) \
+            or crawler_config["max_number_articles_to_get_from_one_seed"] < 0 \
+            or not isinstance(crawler_config["max_number_articles_to_get_from_one_seed"], int) \
+            or crawler_config["total_articles_to_find_and_parse"] < 0:
+        is_crawler_config_ok = False
+        raise IncorrectNumberOfArticlesError
+
+    elif crawler_config["max_number_articles_to_get_from_one_seed"] >= crawler_config["total_articles_to_find_and_parse"]:
+        is_crawler_config_ok = False
+        raise NumberOfArticlesOutOfRangeError
+
+    if is_crawler_config_ok:
+        return (crawler_config["base_urls"], crawler_config["total_articles_to_find_and_parse"],
+                crawler_config["max_number_articles_to_get_from_one_seed"])
 
 
 if __name__ == '__main__':
     # YOUR CODE HERE
-    pass
+    import constants
+    seed_urls, max_articles, max_articles_per_seed = validate_config(constants.CRAWLER_CONFIG_PATH)
```

Comment:
I think crawler_config tests fail, because all tests receive UnknownConfigError from your validate function. It means that you need to rewrite checks a bit. For example: one test is waiting IncorrectNumberOfArticlesError error, but your function gives UnknownConfigError to the test, which is not correct. Actually, all tests receive UnknownConfigError from your valdation function. It means that your first check catches cases of all types and returns the same error for these cases, which is not correct.